src/dashboard/src/components/api/views.py

```diff
@@ -157,23 +182,14 @@ def status(request, unit_uuid, unit_type):
     return helpers.json_response(response)
 
 
+@_api_endpoint(expected_methods=['GET'])
 def waiting_for_user_input(request):
```

Comment:
I wonder how often callers want info about both transfers & SIPs? Does it make sense to have versions of many of these endpoints as unit-agnostic?  We don't actually have an `/api/unit/` or `/unit/` set of endpoints right now - the `unit` component is just functionality common between ingest & transfer that's offered at both `/transfer/` and `/ingest/`

Given time constraints right now though, I'm inclined to document the current behaviour and fix it later.